### PR TITLE
Intro & about 

### PR DESCRIPTION
**Pull request by leenaAhmed**
install :package video-react

### CODE REVIEW

File: src/modules/static-pages/about/components/section/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import styles from  "./style.module.css";
+
+import AboutVideo from "./video/index"
+import {Link} from 'react-router-dom';
+
+
+export default class About extends Component {
+  render() {
+     return (
+      <section  className={styles["about-home-section"]}>
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/index.js]: Use `BEM` methodology ==> `about-home` & `about-home_heading` & `about-home_Info`

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]
1. Resolve merge conflicts and fix errors.

File: src/modules/static-pages/about/components/section/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+             </div>
+            <div className="col-lg-6">
+              <div className={styles['about-Info']}>
+                <p className={styles["about-home-lead"]} >
+                    Was founded in 2012 its dependent to "IEEE EGYPT SECTION" which exists in all universities in Egypt.Our Mission: Raising the Name of Al-Azhar University. Providing the required scientific and practical conditions to prepare 
+                    a successful engineer from the students ,
+                      so they become able to compete in the labor market.
+                </p>
+                <Link to='/about-us'>
+                    <button className={styles["about-home-button"]}> Read More </button>
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/index.js]: The label of the button should be more descriptive to improve accessibility.

Also no need to the button as the `Link` is existed.

File: src/modules/static-pages/about/components/section/style.module.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.about-home-section {
+    margin-top: var( --section-top-spacing);
+    margin-bottom: var(--section-bottom-spacing);
+    background-color: var(--layout-primary);
+ }
+.about-home-section .about-home-Heading{
+    font-size:2rem;
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/style.module.css]: There's a css variable called `--heading-2-font-size` to set the font size of headings.

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]

**Comment by dvmhmdsd** [comment]
Why did u installed `redux` & `resolve-url-loader` ?

**Comment by leenaAhmed** [comment]
> Why did u installed `redux` & `resolve-url-loader` ?

I'm don't sure for installed it, I don't use one of them xD

**Comment by dvmhmdsd** [comment]
> > Why did u installed `redux` & `resolve-url-loader` ?
> 
> I'm don't sure for installed it, I don't use one of them xD

Please don't use any package u will not use.

**Comment by dvmhmdsd** [comment]
Please before any push, be sure that there's no conflicts by pulling first and solve the merge conflicts and remove any error.

File: src/modules/static-pages/about/components/section/style.module.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
+.about-home_section {
+    margin-top: var( --section-top-spacing);
+    margin-bottom: var(--section-bottom-spacing);
+    background-color: var(--layout-primary);
+ }
+.about-home_section .about-home_Heading{
+    font-size:var(--heading-2-font-size);
+    color: var( --layout-secondary);
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/style.module.css:9]: No need to set the color of the heading as it has already been set.

File: src/modules/static-pages/about/components/section/style.module.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    text-align: center;
+}
+
+@media (max-width: 991px) {
+    .about-home_section  .about-home_Info  {
+        padding-top: 3rem;
+    } 
+}
+.about-home_section  .about-home_Info .about-home_lead {
+    color: var( --text-secondary);
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/style.module.css]: No need to set the color of the paragraph as it has already been set, and it should be white not blue.

File: src/modules/static-pages/about/components/section/video/index.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+export default class AboutVideo extends Component {
+  
+  render() {
+  
+     return (
+         <div>
+            <Player src={video} 
+                    poster={poster}          
+             >
+               <BigPlayButton position="center" />
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/video/index.js]: The button gets the focus but no sign for that, try to navigate through the `tab` button. Try to make the button more accessible.

File: src/modules/static-pages/about/components/section/index.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+             </div>
+            <div className="col-lg-6">
+              <div className={styles['about-home_Info']}>
+                <p className={styles["about-home_lead"]} >
+                    Was founded in 2012 its dependent to "IEEE EGYPT SECTION" which exists in all universities in Egypt.Our Mission: Raising the Name of Al-Azhar University. Providing the required scientific and practical conditions to prepare 
+                    a successful engineer from the students ,
+                      so they become able to compete in the labor market.
+                </p>
+                <Link to='/about-us' className={styles["about-home_button"]}>
+                    Read More
```
Comment (dvmhmdsd) [inline src/modules/static-pages/about/components/section/index.js:28]: Please add a descriptive label to the button, as `Read More` is not good for accessibility.

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]

File: src/modules/static-pages/about/components/section/video/index.js (hunk 6 above)
Comment (dvmhmdsd) [reply src/modules/static-pages/about/components/section/video/index.js]: Also, try to make the text of video control bigger

Comment (leenaAhmed) [reply src/modules/static-pages/about/components/section/video/index.js]: I don't know exactly what  I should do I'm not  familiar for this case, anyway I tried to fix it

**Pull request merged**